src/features/notifications/NotificationItem.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { memo } from 'react'
+import { formatDistance, fromUnixTime } from 'date-fns'
+import Box from '../../components/Box'
+import HeliumBubble from '../../assets/images/heliumBubble.svg'
+import Text from '../../components/Text'
+import { Notification } from '../../store/account/accountSlice'
+import TouchableOpacityBox from '../../components/TouchableOpacityBox'
+
+type Props = {
+  notification: Notification
+  isFirst: boolean
+  isLast: boolean
+  onNotificationSelected: (notification: Notification) => void
+}
+const NotificationItem = ({
+  notification,
+  isFirst,
+  isLast,
+  onNotificationSelected,
+}: Props) => {
+  const isNew = !notification.viewed_at
```

Comment:
should we also flag off the green circle in the bottom tab svg unless you have unread notifications?

Reply:
oh, yeah, good call.